pages/advanced/cache.en-US.md

```diff
@@ -91,22 +90,54 @@ In the below example, `matchMutate` can receive a regex expression as key, and b
 
 ```js
 function matchMutate(matcher, data, shouldRevalidate = true) {
-  const keys = [];
+  const keys = []
   if (matcher instanceof RegExp) {
     // `provider` is your cache implementation, for example a `Map()`
     for (const k of provider.keys()) {
       if (matcher.test(k)) {
-        keys.push(k);
+        keys.push(k)
       }
     }
   } else {
-    keys.push(matcher);
+    keys.push(matcher)
   }
 
-  const mutations = keys.map((k) => mutate(k, data, shouldRevalidate));
-  return Promise.all(mutations);
+  const mutations = keys.map((k) => mutate(k, data, shouldRevalidate))
+  return Promise.all(mutations)
 }
 
 matchMutate(/^key-/) // revalidate keys starting with `key-`
 matchMutate('key-a') // revalidate `key-a`
 ```
+
+### Sync Cache to LocalStorage
+
+You might want to sync your cached states to `localStorage` in some special cases, to recover from the persisted state more easily next time while reloading the app.
+
+```js
+function createProvider() {
+  const map = new Map()
+  const provider = {
+    set(key, value) {
+      map.set(key, value)
+      localStorage.setItem(key, JSON.stringify(value))
+    },
```

Comment:
```suggestion
function createProvider() {
  const map = new Map(localStorage.getItem('app-cache') || [])

  window.addEventListener('beforeunload', () => {
    localStorage.setItem('app-cache', map.entries())
  })
```

Reply:
:eyes: did you forget to `JSON.stringify/parse` before putting/getting items from localStorage?